app/main.py

```diff
@@ -18,27 +18,18 @@ async def main() -> None:
     speaker_id = await service.register_device(speaker)
     toilet_id = await service.register_device(toilet)
 
-    async def coroutines(*coroutines) -> None:
-        tasks = [asyncio.create_task(coro) for coro in coroutines]
-        await asyncio.gather(*tasks)
-
-    wake_up_program = [
+    await service.run_program([
         Message(hue_light_id, MessageType.SWITCH_ON),
         Message(speaker_id, MessageType.SWITCH_ON),
         Message(speaker_id, MessageType.PLAY_SONG, "Rick Astley - Never Gonna Give You Up"),
-    ]
+    ])
 
-    sleep_program = [
+    await service.run_program([
         Message(hue_light_id, MessageType.SWITCH_OFF),
         Message(speaker_id, MessageType.SWITCH_OFF),
         Message(toilet_id, MessageType.FLUSH),
         Message(toilet_id, MessageType.CLEAN),
-    ]
-
-    # run the programs
-    await coroutines(service.run_program(wake_up_program),
-                     service.run_program(sleep_program))
-
+    ])
```

Comment:
you must use run_parallel, run_sequence to run programs

Reply:
<img width="815" alt="image" src="https://github.com/mate-academy/py-asyncio-iot/assets/61380547/8fd5c48c-9284-42ca-b4f2-f3597cd4c23a">